spec.html

```diff
@@ -37678,24 +37678,26 @@ <h1>ThenFinally Function</h1>
               1. Let _valueThunk_ be equivalent to a function that returns _value_.
               1. Return ? Invoke(_promise_, `"then"`, &laquo; _valueThunk_ &raquo;).
           </emu-alg>
-      </emu-clause>
+          <p>The `length` property of a ThenFinally function is *1*.</p>
+        </emu-clause>
 
-      <emu-clause id="sec-catchfinallyfunction" aoid="CatchFinallyFunction">
-        <h1>CatchFinally Function</h1>
-        <p>A CatchFinally function is an anonymous built-in function that has a [[Constructor]] and an [[OnFinally]] internal slot. The value of the [[Constructor]] internal slot is a `Promise`-like constructor function object, and the value of the [[OnFinally]] internal slot is a function object.</p>
-        <p>When a CatchFinally function _F_ is called with argument _reason_, the following steps are taken:</p>
-        <emu-alg>
-            1. Let _onFinally_ be _F_.[[OnFinally]].
-            1. Assert: IsCallable(_onFinally_) is *true*.
-            1. Let _result_ be ? Call(_onFinally_, *undefined*).
-            1. Let _C_ be _F_.[[Constructor]].
-            1. Assert: IsConstructor(_C_) is *true*.
-            1. Let _promise_ be ? PromiseResolve(_C_, _result_).
-            1. Let _thrower_ be equivalent to a function that throws _reason_.
-            1. Return ? Invoke(_promise_, `"then"`, &laquo; _thrower_ &raquo;).
-        </emu-alg>
+        <emu-clause id="sec-catchfinallyfunction" aoid="CatchFinallyFunction">
+          <h1>CatchFinally Function</h1>
```

Comment:
Unrelated nit: Seems like similar closures in the spec have headings in plural and with words separated, e.g., [Promise Resolve Functions](http://www.ecma-international.org/ecma-262/#sec-promise-resolve-functions).